tests/find/test-suite-1/test.default-index.js

```diff
@@ -390,4 +390,90 @@ testCases.push(function (dbType, context) {
           });
       });
   });
+
+  it('null values are indexed', function () {
+      var db = context.db;
+      return db.createIndex({
+          index: {
+              fields: ['foo']
+          }
+      }).then(function () {
+          return db.bulkDocs([
+              {_id: '1', foo: null},
+              {_id: '2', foo: 1},
+              {_id: '3', foo: 2},
+              {_id: '4', foo: 3}
+          ]).then(function () {
+              return db.find({
+                  selector: {
+                      foo: {$ne: 1}
```

Comment:
I'm pretty sure CouchDB won't use the foo index for a `$ne`. You need to use another selector on foo to use the index.